moveit_core/robot_state/include/moveit/robot_state/cartesian_interpolator.h

```diff
@@ -75,10 +75,16 @@ struct JumpThreshold
 
 /** \brief Struct for containing max_step for computeCartesianPath
 
-    Setting translation to zero will disable checking for translations and the same goes for rotation */
+    Setting translation to zero will disable checking for translations. The same goes for rotation.
+    Initializing with only one value (translation) sets the rotation such that
+    1 cm of allowed translation = 2 degrees of allowed rotation. */
 struct MaxEEFStep
 {
-  MaxEEFStep(double translation = 0.0, double rotation = 0.0) : translation(translation), rotation(rotation)
+  MaxEEFStep(double translation, double rotation) : translation(translation), rotation(rotation)
+  {
+  }
+
+  MaxEEFStep(double translation) : translation(translation), rotation(3.5 * translation)  // 0.035 rad = 2 deg
```

Comment:
I'd call the parameter `step_size` if it affects both parts.
Any factor >= 1 <= 10 seems reasonable here.

Funny to hear you argue in terms of degree after talking about tau so much ;)

Reply:
True! It makes no sense to call it translation in this case. 

Degrees are just the most convenient unit at this scale, just like millimeters and centimeters. 1 cm is the default in the tutorial, so I based it on that.